build/miengine.settings.targets

```diff
@@ -1,65 +1,46 @@
-  </Target>
-  
+  -->  
   <Import Project="all_projects.settings.targets"/>
   <Import Project="package_versions.settings.targets" />
   <Import Project="version.settings.targets" />
 
   <Choose>
     <When Condition="'$(Lab)' == 'true'">
-        <PropertyGroup>
-            <!--NOTE:DevDivPublicKey.snk comes from src\\tools\\devdiv\\FinalPublicKey.snk -->
-            <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)..\Keys\DevDivPublicKey.snk</AssemblyOriginatorKeyFile>
-
-            <!-- Use PublicSign when TEST_LAB_BUILD is enabled -->
-            <PublicSign Condition="'$(TEST_LAB_BUILD)'!=''">true</PublicSign>
-            <DelaySign Condition="'$(TEST_LAB_BUILD)'==''">true</DelaySign>
-        </PropertyGroup>
-    </When>
-    <Otherwise>
-        <PropertyGroup>
-            <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)..\Keys\ExternalKey.snk</AssemblyOriginatorKeyFile>
-            <DelaySign>false</DelaySign>
-        </PropertyGroup>
-    </Otherwise>
-  </Choose>
-  
-  <Choose>
-    <When Condition="$(Configuration.Contains('Desktop'))">
       <PropertyGroup>
-        <!-- "Desktop.*" configs are used to build the cross-platform bits that are used in the VS Code C++ extension -->
-        <IsXPlat>true</IsXPlat>
-        <DefineConstants>$(DefineConstants);XPLAT</DefineConstants>
+        <!--NOTE:DevDivPublicKey.snk comes from src\\tools\\devdiv\\FinalPublicKey.snk -->
+        <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)..\Keys\DevDivPublicKey.snk</AssemblyOriginatorKeyFile>
+
+        <!-- Use PublicSign when TEST_LAB_BUILD is enabled -->
+        <PublicSign Condition="'$(TEST_LAB_BUILD)'!=''">true</PublicSign>
+        <DelaySign Condition="'$(TEST_LAB_BUILD)'==''">true</DelaySign>
       </PropertyGroup>
     </When>
     <Otherwise>
       <PropertyGroup>
-        <IsXPlat>false</IsXPlat>
+        <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)..\Keys\ExternalKey.snk</AssemblyOriginatorKeyFile>
+        <DelaySign>false</DelaySign>
       </PropertyGroup>
     </Otherwise>
   </Choose>
-
+  
   <!-- Symbol Settings -->
   <Choose>
     <When Condition="$(Configuration.Contains('Debug'))">
       <PropertyGroup>
         <DebugSymbols>true</DebugSymbols>
-        <DebugType>full</DebugType>
+        <DebugType>portable</DebugType>
+        <DebugType Condition="$([MSBuild]::IsOSPlatform('windows')) AND '$(BUILD_BUILDURI)'==''">full</DebugType>
         <DefineConstants>$(DefineConstants);DEBUG;TRACE</DefineConstants>
         <Optimize>false</Optimize>
       </PropertyGroup>
     </When>
     <Otherwise>
       <!-- Release Symbol Settings -->
       <PropertyGroup>
-        <DebugType>pdbonly</DebugType>
+        <DebugType>portable</DebugType>
+        <DebugType Condition="$([MSBuild]::IsOSPlatform('windows')) AND '$(Lab)' == 'true'">full</DebugType>
```

Comment:
>full [](http://example.com/codeflow?start=92&length=4)

@WardenGnaw I am not sure if it makes a difference, but I think you want `pdbonly`